OneMore/Commands/File/ImportCommand.cs

```diff
@@ -600,6 +603,7 @@ private async Task ImportMarkdownFile(string filepath, CancellationToken token)
 
 					converter = new MarkdownConverter(page);
 					converter.RewriteHeadings();
+					converter.RewriteTodo();
```

Comment:
See my comment on this method. Should not affect entire page. I'm removing this line.